aire/src/navbar/navbar.pug

```diff
@@ -1,3 +1,3 @@
 template
-    nav.uk-navbar-container.uk-navbar-sticky.uk-dark(ref="element")
+    nav.uk-navbar-container.uk-navbar-sticky.uk-dark(ref="element" uk-navbar)
```

Comment:
these pseudo-elements are only used when initializing the entire page, which we don't want to do.